README.md

```diff
@@ -1 +1,136 @@
-# NFT-Stats
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name : Crypto Stats
+> This project consume real live crypto data from the [CoinCap API](docs.coincap.io/). It then lists 200 top coins with clickable link to view historical data in a beautifully designed line graph.
+> The application only shows data for the past 30 days
+
+## Built With
+> React, Redux and CSS for styling
+
+## [Live Demo](https://crypto-stats-app.herokuapp.com)
```

Comment:
- [ ]  You did a great job, but in your videos, you forgot to highlight a piece of the code you find interesting or proud of making. Please follow  the mentioned guideline while you are making the vidoes